packages/apps/hr/src/components/molecules/ExpenseLineDetailCard/ExpenseLineDetailCard.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {useThemeColor, CardIconButton} from '@axelor/aos-mobile-ui';
+import {useTranslator} from '@axelor/aos-mobile-core';
+import {Expense, ExpenseLine} from '../../../types';
+import {View} from 'react-native';
+import {ExpenseLineCard} from '../../atoms';
+import {StyleSheet} from 'react-native';
```

Comment:
Clean imports please : 

- react
- react-native
- react librairies
- our packages
- other imports

It makes it easier to understand or the search something